src/lava/magma/core/process/process.py

```diff
@@ -333,8 +333,10 @@ def run(self,
 
         Parameters
         ----------
-        condition : AbstractRunCondition
-            Specifies for how long to run the Process.
+        condition : Optional[Union[AbstractRunCondition, int]], default=None
```

Comment:
Just had this thought: When we have a Process with a `PyProcModel` that implements the `AsyncProtocol`, are we treating an iteration of the `run()` method as a "timestep"? I'm generally confused about these global time steps of the `RunSteps` condition when dealing with a network of Processes that run asynchronously.